static/app/views/alerts/list/rules/row.tsx

```diff
@@ -57,7 +59,24 @@ function RuleListRow({
   const [assignee, setAssignee] = useState<string>('');
   const activeIncident = hasActiveIncident(rule);
 
-  function renderLastIncidentDate(): React.ReactNode {
+  const isActivatedAlertRule =
+    !isIssueAlert(rule) && rule.monitorType === MonitorType.ACTIVATED;
+
+  // TODO(davidenwang): Decompose this further
```

Comment:
not sure how i feel about this yet, but i think my idea would be to refactor it out similar to the `CombinedAlertBadge` and make the logic here clearer since it feels a bit fragmented

Reply:
https://github.com/getsentry/sentry/pull/75309 ended up doing it here